### ISSUE TITLE
Incompatible as an es5 package

### ISSUE THREAD

**Issue by zeachco** [caub/roman-numerals#1]
Hello,
as most bundle system don't transpile node module (for performance reasons) this might get unsupported on some browsers or break some webpack build systems that still use uglify.js

here is an example of code that breaks in an es5 env:
https://github.com/caub/roman-number/blob/74a2620a478670e572d532628c71b384f4f72e78/index.js#L63

Was it planned to expose a build that is already compatible?

meanwhile the quick fix is to add this repo's path to the babel transpilation and it works fine so it's not critical.

**Comment by caub** [comment]
that's right, it needs rollup or changing to function, if you've time you can make a PR, else I may do it later

you can also use uglify-es too, if you don't care about ie <10, which is quite safe

**Comment by zeachco** [comment]
`uglify-es` is even a better option, good call :confetti_ball: 
I realised it's not just an arrow function that breaks, I'll probably check in depth this weekend.

Thanks :wave:
